flang/lib/Evaluate/wrappers.c

```diff
@@ -0,0 +1,25 @@
+//===-- lib/Evaluate/wrappers.c -------------------------------------------===//
```

Comment:
Does this code absolutely have to be in C rather than C++?  It might be strange to have an effectively empty source file on hosts other than AIX.

Reply:
I can make it C++ and I can use something like this to get around the `complex` support.
```
#ifdef __clang_major__
#pragma clang diagnostic ignored "-Wc99-extensions"
#endif
```

>It might be strange to have an effectively empty source file on hosts other than AIX.

I agree it is strange. If I let the file compiled for other hosts, I need to add something for MSVC compile. However, it looks strange as well because the code should never be called on the windows platform.